src/server_manager/web_app/ui_components/outline-gcp-create-server-app.ts

```diff
@@ -303,32 +307,57 @@ export class GcpCreateServerApp extends LitElement {
     } else {
       if (!this.billingAccounts || this.billingAccounts.length === 0) {
         this.showBillingAccountSetup();
+        // Check every five seconds to see if an account has been added.
+        this.billingAccountsRefreshLoop = window.setInterval(() => {
+          this.refreshBillingAccounts();
+        }, 5000);
       } else {
         this.showProjectSetup(this.project);
       }
     }
   }
 
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.stopRefreshingBillingAccounts();
+  }
+
   private init() {
     this.currentPage = '';
     this.selectedProjectId = '';
     this.selectedBillingAccountId = '';
+    this.stopRefreshingBillingAccounts();
   }
 
   private showBillingAccountSetup(): void {
     this.currentPage = 'billingAccountSetup';
   }
 
-  private async handleBillingVerificationNextTap(): Promise<void> {
-    this.showProjectSetup();
+  private async refreshBillingAccounts(): Promise<void> {
+    this.billingAccounts = await this.account.listBillingAccounts();
+    // FIXME: listBillingAccounts() can reject, resulting in an uncaught
+    // exception here that is shown in the debug console but not reflected
+    // in the UI.
+
+    if (this.billingAccounts && this.billingAccounts.length > 0) {
+      this.stopRefreshingBillingAccounts();
+      this.showProjectSetup();
+      window.bringToFront();
+    }
   }
 
-  private async showProjectSetup(existingProject?: Project): Promise<void> {
-    this.billingAccounts = await this.account.listBillingAccounts();
-    if (!this.billingAccounts || this.billingAccounts.length === 0) {
-      return this.showBillingAccountSetup();
+  private stopRefreshingBillingAccounts(): void {
+    if (this.billingAccountsRefreshLoop !== null) {
```

Comment:
nit: you can `clearInterval(null)` and it will just do nothing.

Reply:
Done